src/frontend/efiling-frontend/src/domain/package/package-review/tooltip/Tooltip.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { UncontrolledTooltip } from "reactstrap";
+import "./Tooltip.scoped.css";
+import "./Tooltip.css";
+
+export default function BcGovTooltip({ content, styling, target }) {
```

Comment:
Very nice. If this BCGovToolTip can be used in other components, I might suggest to move this to /src/components - I think this where we will be putting common components.
Yeah, I should do the same for a BCGovModal common component.

Reply:
That's a good call, I can make it a little more extensible and move it into components. At the moment its more designed specifically for the withdraw icon. Wouldn't take long though

Reply:
K.  Lets do that in a separate PR to keep them smaller.
I'll do the same for a BCGovModal.